plugins/chain_plugin/account_query_db.cpp

```diff
@@ -476,13 +478,31 @@ namespace eosio::chain_apis {
          return result;
       }
 
+      account_query_db::get_all_accounts_result 
+      get_all_accounts( ) {
+         std::lock_guard<std::mutex> lock(controller.acct_obj_mutex());
+
+         using result_t = account_query_db::get_all_accounts_result;
+         result_t result;
+
+         using acct_obj_idx_type = chainbase::get_index_type<chain::account_object>::type;        
+         const auto& accts = controller.db().get_index<acct_obj_idx_type >().indices().get<chain::by_name>();
+
+         for (auto& a : accts) {
```

Comment:
This should time out after 10ms and provide a `more` like other chain api calls.